R/package.R

```diff
@@ -168,12 +164,9 @@ check_bootstrap_version <- function(version, pkg) {
     cli::cli_warn("{.var bootstrap: 4} no longer supported, using {.var bootstrap: 5} instead")
     5
   } else {
-    msg_fld <- pkgdown_field(pkg, c("template", "bootstrap"), cfg = TRUE, fmt = TRUE)
-    cli::cli_abort(
-      c(
-        "Boostrap version must be 3 or 5.",
-        x = paste0("You set a value of {.val {version}} to ", msg_fld, ".")
-      ),
+    config_abort(
+      pkg,
```

Comment:
pkg is not properly passed here. Try in the pkgdown repo to set bootstrap: 7, and I get

```r
Error in `fun(..., .envir = .envir)`:
! Could not evaluate cli `{}` expression: `config_path(pkg)`.
Caused by error in `pkg$src_path`:
! $ operator is invalid for atomic vectors
```

Reply:
Good catch, thanks!